mteb/evaluation/evaluators/PairClassificationEvaluator.py

```diff
@@ -45,8 +45,10 @@ def __init__(
 
         assert len(self.sentences1) == len(self.sentences2)
         assert len(self.sentences1) == len(self.labels)
-        for label in labels:
-            assert label == 0 or label == 1
+
+        # removing this indic xnli is 3-way (entailment, neutral, )
+        # for label in labels:
+        #     assert label == 0 or label == 1
```

Comment:
Undo this change and filter out the neutral values just as it has ben done here: [XNLI](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/PairClassification/multilingual/XNLI.py)